src/utils/data_validation_test.ts

```diff
@@ -54,15 +54,18 @@ export function isShared(
 	if (!file || file.extension !== "md" || meta === null) {
 		return false;
 	}
-	const folderList = settings.plugin.excludedFolder;
+	const excludedFolderList = settings.plugin.excludedFolder;
 	const shareKey = otherRepo ? otherRepo.shareKey : settings.plugin.shareKey;
+	if (shareKey === "" && !file.basename.startsWith("DRAFT")) {
+		return true;
+	}
```

Comment:
This is the relevant part. Rest is just a var rename.

Reply:
Hello !
It's a good idea, but I have another idea for this suggestion. To prevent confusing user and add more versatility, I will add settings for that!
Wait for it :)

Reply:
Thank you.
I have currently identified how sheer existence of frontmatter interferes with another script I am trying to incorporate into my workflow and shared an issue with creator of the other script: https://github.com/themcaffee/GoSquatch/issues/4

I am going to try to fix this with the GoSquatch creator, as frontmatter should be able to exist, but FYI.